### PR TITLE
display precision of galactic coordinates

### PR DESCRIPTION
**Pull request by guynir42**
increased display precision of galactic coordinates in sources/candidates pages. Fixes issue #809

### CODE REVIEW

**Comment by guynir42** [comment]
Hi @dannygoldstein @acrellin this is a minor bug fix so I can just merge it, but maybe you want to make sure it looks ok...

**Review by acrellin — APPROVED** [review]

File: static/js/components/Candidate.jsx [hunk 1]
```diff
@@ -56,8 +56,8 @@ const Candidate = ({ route }) => {
         &nbsp; (&alpha;,&delta;=
         {ra_to_hours(candidate.ra)}, &nbsp;
         {dec_to_hours(candidate.dec)}) &nbsp; (l,b=
-        {candidate.gal_lon.toFixed(1)}, &nbsp;
-        {candidate.gal_lat.toFixed(1)}
+        {candidate.gal_lon}, &nbsp;
```
Comment (dannygoldstein) [inline static/js/components/Candidate.jsx:59]:
```suggestion
        {candidate.gal_lon.toFixed(6)}, &nbsp;
```

File: static/js/components/Candidate.jsx [hunk 2]
```diff
@@ -56,8 +56,8 @@ const Candidate = ({ route }) => {
...
+        {candidate.gal_lat}
```
Comment (dannygoldstein) [inline static/js/components/Candidate.jsx:60]:
```suggestion
        {candidate.gal_lat.toFixed(6)}
```

File: static/js/components/Source.jsx [hunk 3]
```diff
@@ -99,8 +99,8 @@ const Source = ({ route }) => {
         &nbsp; (&alpha;,&delta;=
         {ra_to_hours(source.ra)}, &nbsp;
         {dec_to_hours(source.dec)}) &nbsp; (l,b=
-        {source.gal_lon.toFixed(1)}, &nbsp;
-        {source.gal_lat.toFixed(1)}
+        {source.gal_lon}, &nbsp;
```
Comment (dannygoldstein) [inline static/js/components/Source.jsx:102]:
```suggestion
        {source.gal_lon.toFixed(6)}, &nbsp;
```

File: static/js/components/Source.jsx [hunk 4]
```diff
@@ -99,8 +99,8 @@ const Source = ({ route }) => {
...
+        {source.gal_lat}
```
Comment (dannygoldstein) [inline static/js/components/Source.jsx:103]:
```suggestion
        {source.gal_lat.toFixed(6)}
```

**Review by dannygoldstein — CHANGES_REQUESTED** [review]

**Review by dannygoldstein — APPROVED** [review]

**Pull request merged**